config/locales/views/talk/show/en.yml

```diff
@@ -0,0 +1,7 @@
+# See https://github.com/svenfuchs/rails-i18n/tree/master/rails%2Flocale for starting points.
+
+en:
+  talk:
+    show:
+      welcome: "Hello Everyone! Welcome to Publiclab Talk Pad"
```

Comment:
Hi, @jiteshjha - if you could ask stevie and liz (each @publiclab.org) via email about this, that'd be great.